### PR TITLE
Changes made

### LINKED ISSUE #115: UI/UX Readme list of improvements
### Idea Contribution

- [x] I have read all the feature request issues.

### Explain feature request

I identified some areas of improvement within the UI/UX Category, as listed below by section:

*  **Tutorials and courses:** 
    - Could be beneficial to add a column to indicate if it's a paid or free resource. For some resources it's clarified in the title, but for consistency with the tables below it would be better to add that info into a column.

* **Tools:**
    - Several links are broken or the resource exists under a new URL. Need to update/remove them.
    - Tools of the same provider could be grouped together to facilitate access (e.g. Adobe tools)
  
* **Design inspiration:**
    - Broken links need to be updated/removed

* **Books:**
    - Different editions or revisits of the same books should be grouped together for easy identification

* **Communities:**
    - Adding a column to indicate in which platform you can find the community (Reddit, LinkedIn, Facebook, etc) can boost clarity.
    - After checking each URL some communities are really inactive, to keep the resource updated it would be good removing them from the list.
        - https://www.linkedin.com/groups/4496205/ - 0 publications a day
        - https://www.linkedin.com/groups/1854192/ - 0 publications a week
        - Last two links of the list in the document are described as different resources but lead to the same Linkedin group. 

* **Interview:**
    - Changing the title to something more descriptive would be good for clarifying what the section is about. I suggest something like "UX/UI Job interview resources" because that implies that we are talking about Job interviews vs any other kind of interviews and also that there are several types of resurces that you can find in that section, like videos, podcasts, articles, etc.

* **Newsletter:**
    - Broken links need to be updated/removed

### Explain your solution

_No response_

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Divya-1612-reddy**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Fixes Issue

#closes #115

### Description: 

"I added the prices, and working links, and simplified the description. in the books section and added the whether it is a free or paid course in tutorials or courses"
### Resource URL: 

https://github.com/Divya-1612-reddy/ResourceHub/tree/main/UI-UX%20Design

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [ ] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [ ] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: UI-UX Design/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # UI & UX Design 🎨
 
-## Table of Contents
+## Navigator
```
Comment (jfmartinz) [inline UI-UX Design/README.md:3]:
```suggestion
## Table of Contents
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Please change this before I merged it, thank you

File: UI-UX Design/README.md (hunk 1 above)
Comment (Divya-1612-reddy) [reply UI-UX Design/README.md]: yes i changed it and sorry for the delay

**Comment by Divya-1612-reddy** [comment]
i changed  now u can merge it

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Oh I forgot, you actually changed the [book](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#books), can you please also add Cost in the [Tutorial or Courses](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#tutorials-or-courses) as well.

This is the first issue. Change this and I will merged this PR, thank you for your patience

**Comment by Divya-1612-reddy** [comment]
done

**Review by jfmartinz — APPROVED** [review]
Thanks @Divya-1612-reddy.

**Pull request merged**